lib/src/browser.dart

```diff
@@ -89,11 +89,15 @@ class _Chrome extends Browser {
   static Version _getVersion(NavigatorProvider navigator) {
     Match match = new RegExp(r"Chrome/(\d+)\.(\d+)\.(\d+)\.(\d+)\s")
         .firstMatch(navigator.appVersion);
```

Comment:
ok, I just went this route b/c it looked like a similar path was used later on in this file https://github.com/Workiva/platform_detect/blob/077e8107c8fd696ec786fb9fc2057cdceac9137f/lib/src/browser.dart#L174